packages/protocol/contracts/bridge/libs/LibBridgeProcess.sol

```diff
@@ -78,11 +78,10 @@ library LibBridgeProcess {
 
         // We retrieve the necessary ether from EtherVault
         address ethVault = resolver.resolve("ether_vault", false);
-        if (ethVault != address(0)) {
-            EtherVault(payable(ethVault)).releaseEther(
-                message.depositValue + message.callValue + message.processingFee
-            );
-        }
+        EtherVault(payable(ethVault)).releaseEther(
```

Comment:
Good catch, but I think we should have the above change as we do allow the ethVault to be address(0)